src/js/dropdown-menu/specs/dropdown-menu-view-spec.js

```diff
@@ -256,21 +257,33 @@ define([
                     closeOnKeypressTest(constants.keyCodes.space);
                 });
 
-                it('should return to the button after pressing down arrow key while at bottom of dropdown menu', function() {
-                    focusTrapDown(constants.keyCodes.down, 4);
-                });
+                it(
```

Comment:
Are these lines too long? My preference is to have a shorter message, and keep everything on one line. I like tests to read concisely.